plugins/modules/route53.py

```diff
@@ -136,13 +137,13 @@
   failover:
     description:
       - Failover resource record sets only. Whether this is the primary or
-        secondary resource record set. Allowed values are PRIMARY and SECONDARY
-      - Mutually exclusive with I(weight) and I(region).
+        secondary resource record set. Allowed values are V(PRIMARY) and V(SECONDARY).
+      - Mutually exclusive with O(weight) and O(region).
     type: str
     choices: ['SECONDARY', 'PRIMARY']
   vpc_id:
     description:
-      - "When used in conjunction with private_zone: true, this will only modify records in the private hosted zone attached to this VPC."
+      - "When used in conjunction with O(private_zone=true), this will only modify records in the private hosted zone attached to this VPC."
```

Comment:
```suggestion
      - When used in conjunction with O(private_zone=true), this will only modify records in the private hosted zone attached to this VPC.
```